src/replication.c

```diff
@@ -1426,7 +1473,7 @@ initialize_volume(spec_t *spec)
 	TAILQ_INIT(&spec->rq);
 	TAILQ_INIT(&spec->rwaitq);
 
-        if(init_mempool(&spec->rcommon_deadlist, rcmd_mempool_count, sizeof (rcmd_t), 0,
+        if(init_mempool(&spec->rcommon_deadlist, rcmd_mempool_count, 0, 0,
```

Comment:
default seems to be SP/SC for ring.. it need to be changed according to the ring usage (for all rings)

Reply:
ok.. we are using MP/MC.. its fine then